palindrome.c

```diff
@@ -0,0 +1,53 @@
+/* Can Parlar - scans a string and checks if it is a palindrome word or not
+*/
+#include <stdio.h> 
```

Comment:
your code unfortunately doesn't handle many of the test cases: 
1. empty string - this is a palindrome. 
2. single letters - "a" or "d" or one letter of the alphabet is a palindrome.
3. numbers - 121 is a palindrome
4. capitals - "Hannah" is a palindrome

You should  thoroughly test your code to get at these test and edge cases.